fern/versions/public.yml

```
@@ -97,7 +91,19 @@ navigation:
             path: ../docs/pages/plug-sdk/methods.mdx
           - page: Custom implementation of PLuG widget
             slug: customize
-            path: ../docs/pages/plug-sdk/customize.mdx  
+            path: ../docs/pages/plug-sdk/customize.mdx
+      - section: Mobile SDK
+        slug: mobile
+        contents:
+          - page: Integrate PLuG SDKs with your Android app
```

"Android app" is more important than "Integrate PLuG SDKs" so put it first.
```suggestion
          - page: Android app integration
```

---

Updated